<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWindow</class>
 <widget class="QMainWindow" name="ChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="showScroll">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>551</width>
      <height>241</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>549</width>
       <height>239</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="peerLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>561</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Name &amp; IP Address</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="sendScroll">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>361</width>
      <height>131</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>359</width>
       <height>129</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="msgBtn">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>320</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fileBtn">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>380</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Send File</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
